lib/modules/platform/azure/readme.md

```diff
@@ -91,6 +98,45 @@ module.exports = {
 
 For the `repositories` key, replace `YOUR-PROJECT/YOUR-REPO` with your Azure DevOps project and repository.
 
+### Using Azure DevOps internal API for pipeline tasks versions
+
+<!-- prettier-ignore -->
+!!! info
+    Renovate bot is now using the set of APIs that azure provides to query the azure-pipelines tasks versions directly from the instance. ([PR #32966](https://github.com/renovatebot/renovate/pull/32966) and [Discussion #24820](https://github.com/renovatebot/renovate/discussions/24820))
+
+To use it, you need to have in your config :
+
+- `platform` = `azure`
+- `endpoint` = `$(System.CollectionUri)` (an [azure predefined variable](https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml))
+- `hostRules.hostType` = `azure-pipelines-tasks`
+
+An example configuration file, in `.json` format this time, would be :
+
```

Comment:
```suggestion
```

Let's use a codeblock title instead of this line. :wink: 

[Material for MkDocs, adding a title to a codeblock](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-a-title)